install.sh

```diff
@@ -110,6 +101,21 @@ main() {
                 "You can find more information about AUR helpers at https://wiki.archlinux.org/title/AUR_helpers"
         fi
 
+    elif available yum; then
+        available yum-config-manager || show $sudo yum install yum-utils -y
+        show $sudo yum-config-manager -y --add-repo https://brave-browser-rpm-release.s3.brave.com/brave-browser.repo
+        show $sudo yum install brave-browser -y
+
+    elif available zypper; then
```

Comment:
I'd put `zypper` before `yum` in case `yum` is around on an OpenSUSE machine.

Reply:
> Also, keep in mind that most dnf installations frequently (usually?) come with a yum symlink, so it's best to think of dnf and yum as a common combo.

That was basically my concern. If `yum` exists on OpenSUSE, then that's not the ideal way to install Brave on that distro. Similar to how `dnf` is better on Fedora. We should only use `yum` if `zypper` or `dnf` are not there IMO.